### PR TITLE
Restore support for RTL XL

### PR DESCRIPTION
**Pull request by whizzoo**
FYI: I made the original Livestreamer plugin as well. Recently, RTL made some changes to the website and the plugin stopped working. These changes make the plugin function again.

### CODE REVIEW

**Comment by whizzoo** [comment]
Hi, what's the reason this update has not been accepted yet? The previous version of this plugin is no longer functional at all...

**Comment by gravyboat** [comment]
@whizzoo Sorry about that it probably just fell through the cracks. I might have opened it then closed the incorrect tab or something, thanks for fixing this up!

File: src/streamlink/plugins/rtlxl.py [hunk 1]
```diff
@@ -1,24 +1,25 @@
...
-_url_re = re.compile(r"""http(?:s)?://(?:\w+\.)?rtlxl.nl/#!/(?:.*)/(?P<uuid>.*?)\Z""", re.IGNORECASE)
-
+_url_re = re.compile("""http(?:s)?://(?:\w+\.)?rtl.nl/video/(?P<uuid>.*?)\Z""", re.IGNORECASE)
 
 class rtlxl(Plugin):
     @classmethod
     def can_handle_url(cls, url):
         return _url_re.match(url)
-
+
```
Comment (gravyboat) [inline src/streamlink/plugins/rtlxl.py]: I should have requested this be modified to remove that extra space but whatever.

**Comment by whizzoo** [comment]
@gravyboat No problem! Thanks for the merge!

**Comment by gravyboat** [comment]
@whizzoo No problem, if you have time do you want to fix line 14 so it's just a blank line instead of those extra spaces? If not I'll try to get around to it. Thanks again.

**Comment by whizzoo** [comment]
@gravyboat Sure thing, I just submitted a new PR for the fixed file.

**Pull request merged**